src/Movie.js

```diff
@@ -0,0 +1,115 @@
+import { useEffect, useState } from 'react';
+import MovieSearch from'./components/MovieSearch'
+import {
+  HeaderMovie,
+  DisplayImages,
+  SubMainCard,
+  ButtonShowMore,
+  MovieTitle,
+  SubContain,
+  SizingImg,
+} from "./styles/Card.styed";
+import{  ErrorCont,
+  ErrorMsg,
+  MainErrorDiv,
+  ErrorImg,}from "./styles/ErrorFound.styed"
+import { useHistory } from "react-router-dom";
+import NotFond from './images/notfound.png'
+
+
+//Api
+const auth="api_key=16f5a39714623a8f11910f456e72441e"
```

Comment:
Add the api endpoints and secrets in the environment variables. Please refer - [Using environment variables in create react app](https://create-react-app.dev/docs/adding-custom-environment-variables/)